pyntc/devices/ios_device.py

```diff
@@ -685,6 +685,109 @@ def file_copy_remote_exists(self, src, dest=None, file_system=None):
         log.debug("Host %s: File %s does not already exist on remote.", self.host, src)
         return False
 
+    def file_download(self, url, dest="", md5=None, file_system=None, read_timeout=2000):
```

Comment:
Should we just do `**netmiko_kwargs` too?  Incase you need `read_timeout_override` or any of the other netmiko kwargs.